client/src/components/PanelDrawer/PanelDrawer.js

```diff
@@ -18,6 +18,8 @@ export default function PanelDrawer({
   actions,
   onClose,
 }) {
+  const percentWidth =
```

Comment:
This seems awkward. Wish we had typing here to enforce my expectation of `percentWidth` always being in string format (with %, px, etc unit)